toolbox/strings.py

```diff
@@ -102,6 +110,70 @@ def is_invite(string: str, *, fullmatch: bool = True) -> bool:
     return False
 
 
+class MarkdownFormat(IntFlag):
+    NONE = 0
+    STRIKETHROUGH = 1
+    ITALIC_UNDERSCORE = 2
+    ITALIC_ASTERISK = 4
+    BOLD = 8
+    UNDERLINE = 16
+    CODE_BLOCK = 32
+    MULTI_CODE_BLOCK = 64
+    QUOTE = 128
+    MULTI_QUOTE = 256
+    SPOILER = 512
+    ALL = (
+        STRIKETHROUGH
+        | ITALIC_UNDERSCORE
+        | ITALIC_ASTERISK
+        | BOLD
+        | UNDERLINE
+        | CODE_BLOCK
+        | MULTI_CODE_BLOCK
+        | QUOTE
+        | MULTI_QUOTE
+        | SPOILER
+    )
+
+
+format_dict = {
```

Comment:
This is a constant so it should be at the top of the file and SCREAMING_SNAKE_CASE